wiki/Tournaments/oSEAT/2/en.md

```diff
@@ -97,38 +97,31 @@ The osu! South East Tournament 2 is run by various community members.
 
 -----------------------
 
-## Match Schedules: Finals
+## Match Schedules: Grand Finals
 
-### Friday, 18 January 2019
+### Saturday, 26 January 2019
 
 | Match Code | Player A |  |  | Player B | Match Time (UTC+8) |
 | :-: | --: | :-: | :-: | :-- | :-: |
-| MB | Crezz | ![][flag_ID] | ![][flag_SG] | GSBlank | **22:00** |
+| LB | Rexeez | ![][flag_ID] | ![][flag_MY] | GSBlank | **21:00** |
 
-### Saturday, 19 January 2019
-
-| Match Code | Player A |  |  | Player B | Match Time (UTC+8) |
-| :-: | --: | :-: | :-: | :-- | :-: |
-| MA | Raindrop | ![][flag_SG] | ![][flag_SG] | SeeL | **22:00** |
-
-### Sunday, 20 January 2019
+### Sunday, 27 January 2019
 
 | Match Code | Player A |  |  | Player B | Match Time (UTC+8) |  |
 | :-: | --: | :-: | :-: | :-- | :-: | :-: |
-| LA | Rexeez | ![][flag_ID] | ![][flag_ID] | Skydiver | **13:30** |  |
-| NA-1 | Raindrop | ![][flag_SG] | ![][flag_MY] | Crezz | **13:30** | ¹ |
-| NA-2 | Raindrop | ![][flag_SG] | ![][flag_MY] | GSBlank | **22:00** | ¹ |
-| NA-3 | SeeL | ![][flag_SG] | ![][flag_MY] | Crezz | **13:30** | ¹ |
-| NA-4 | SeeL | ![][flag_SG] | ![][flag_MY] | GSBlank | **22:00** | ¹ |
+| LC-1 | Skydiver | ![][flag_ID] | ![][flag_ID] | Rexeez | **21:00** | ¹ |
```

Comment:
these footnotes should probably go after the match code, rather than in their own cell

Reply:
That would be a disaster for the readers and to look at.

Reply:
"LC-1 ¹"?

if you mean that the superscript could be confused with the match number, adding a space like above seems to fix the issue

Reply:
also you don't need to make it a numeral, it could just be an asterisk I think

Reply:
> 
> also you don't need to make it a numeral, it could just be an asterisk I think

Don't, last week i have 16 superscript.  And you don't want 16 asterisks filling a cell.

Reply:
well in this case you've only got one :wink:

Reply:
but consistency matters dad, even though match schedules only have to be filled when the tournament's still running.

Reply:
Alongside the asterisks, will apply the suggestion